cime_config/machines/cmake_macros/crayclang-scream_frontier-scream-gpu.cmake

```diff
@@ -5,7 +5,7 @@ set(SCC "cc")
 set(SCXX "hipcc")
 set(SFC "ftn")
 
-string(APPEND CPPDEFS " -DLINUX")
+string(APPEND CPPDEFS " -DLINUX -DSCREAM_SYSTEM_WORKAROUND=1")
```

Comment:
I don't think you ever use the case where it's 0, so prob you could just do `-DSCREAM_SYSTEM_WORKAROUND`, and remove `&& (SCREAM_SYSTEM_WORKAROUND == 1)` from all CPP checks.

Reply:
> I don't think you ever use the case where it's 0, so prob you could just do `-DSCREAM_SYSTEM_WORKAROUND`, and remove `&& (SCREAM_SYSTEM_WORKAROUND == 1)` from all CPP checks.

According to @ambrad, the purpose for the integer complication is as follows. In the future one can imagine needing separate workarounds for Aurora and Frontier. We might then say that Frontier is assigned 1 and Aurora is assigned 2.

Reply:
Makes sense, thanks!